README.md

```diff
@@ -25,15 +25,15 @@ RamaLama eliminates the need to configure the host system by instead pulling a c
 
 ## Accelerated images
 
-| Accelerator             | Image                      |
-| :-----------------------| :------------------------- |
-|  CPU, Vulkan, Apple     | quay.io/ramalama/ramalama  |
-|  HIP_VISIBLE_DEVICES    | quay.io/ramalama/rocm      |
-|  CUDA_VISIBLE_DEVICES   | quay.io/ramalama/cuda      |
-|  ASAHI_VISIBLE_DEVICES  | quay.io/ramalama/asahi     |
-|  INTEL_VISIBLE_DEVICES  | quay.io/ramalama/intel-gpu |
-|  ASCEND_VISIBLE_DEVICES | quay.io/ramalama/cann      |
-|  MUSA_VISIBLE_DEVICES   | quay.io/ramalama/musa      |
+| Accelerator                       | Image                      |
+| :---------------------------------| :------------------------- |
+|  GGML_VK_VISIBLE_DEVICES (or CPU) | quay.io/ramalama/ramalama  |
```

Comment:
Would it be simpler to just name this VULKAN_VISIBLE_DEVICES?

Reply:
It wouldn't because these env var names are all from llama.cpp